05.wc/wc.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+option = ARGV.getopts('lwc')
+filename = ARGV
+
+def standard_input(option)
+  words_width = 7
+  gets_file = readlines
+  gets_file = gets_file.join('')
+
+  chars_in_file = gets_file.bytesize.to_s
+  lines_in_file = gets_file.lines.length.to_s
+  words_in_file = gets_file.split.count.to_s
+  option_values = { 'c' => chars_in_file, 'l' => lines_in_file, 'w' => words_in_file }
+  option_select = option.select { _2 == true }
```

Comment:
```suggestion
  option_select = option.select { _2 }
```

「xが真であること」は`x == true`ではなく`x`だけでOKです